docs/reference/release-notes/highlights.asciidoc

```diff
@@ -106,6 +106,25 @@ search features like queries and aggregations.
 
 {es-pull}88694[#88694]
 
+[discrete]
+[[synthetic_source_technical_preview]]
+=== Synthetic `_source`
+preview::[]
+When Elasticsearch ingests documents, it creates several data structures that
+enable querying and aggregating the data. One of those data structures is called
+`_source`.  `_source` holds the fields of every document. It uses a lot of space
+and is used for specific needs, like reindexing and presenting the document in
+Kibana Discover. 
+
+This release introduces a new feature in technical preview called synthetic
+`_source`. Synthetic `_source` can significantly reduce the index size by, for
+specific data types, recreating the `_source` from doc values. This is useful
+when you don't need to preserve the original structure of the `_source`. While
+doc values don't preserve the original structure of the `_source`, synthetic
+`_source` still allows for capabilities like reindexing.
```

Comment:
I think the critical thing you don't mention is that we need to store the doc values anyway. I tend to explain synthetic _source like this:

Elasticsearch can store 3 things, which have three primary use cases: a search index for finding the documents, a column store called doc_values for aggregating the documents, and _source for returning the precise document you sent us. A lot of people don't need to get the precise document that they send us back, but the whole ecosystem around Elasticsearch relies on being able to get it. So synthetic _source let's us rebuild a look-alike on the fly. It doesn't preserve the precise order of things, but that's *fine* for lots of people. Like I said, lots of people never even look at `_source`. It does take a little longer to fetch, but that performance doesn't effect aggregations at all, which tend to be the bigger performance bottleneck most folks face.

Reply:
@nik9000 I've pushed a change that includes the changes you've suggested. Let me know what you think.